services/mapit/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.7'
+  
+volumes:
+  mapit-pg:
+
+x-mapit: &mapit
+  build:
+    context: .
+    dockerfile: services/mapit/Dockerfile
+  image: mapit
+  working_dir: /mapit
+
+services:
+  mapit-pg:
+    image: mdillon/postgis:9.6-alpine
+    volumes:
+      - mapit-pg:/var/lib/postgresql/data
+
+  mapit:
+    <<: *mapit
+    volumes:
+        - ~/govuk/mapit:/mapit
```

Comment:
Can we make this consistent with [how we do this in every other service](https://github.com/alphagov/govuk-docker/blob/master/services/content-publisher/docker-compose.yml#L9)?